components/nodemanager-service/Makefile

```diff
@@ -180,7 +180,7 @@ test-integration:
 
 run-db-tests:
 	@POSTGRES_URI="$(POSTGRES_URI)" CGO_ENABLED=0 \
-		go test -v -cover -database ./pgdb/...
+		go test -v ./pgdb/... -cover -database -parallel=1 -p 1 -failfast
```

Comment:
to be honest, i didn't take the time to look up all these different flags and what they mean. just copied from compliance makefile.

Reply:
i also don't understand why this tests things when the previous command did not. is it the order of things??

Reply:
This seems really weird to me. It looks like the original command should work:

```
Petes-MacBook-Pro:compliance-service pete$ go help test
usage: go test [build/test flags] [packages] [build/test flags & test binary flags]
```

Maybe it was working and then one of the go version upgrades changed how this worked?